res/TensorFlowLiteRecipes/Net_Sqrt_Div_000/test.recipe

```diff
@@ -0,0 +1,54 @@
+operand {
+  name: "ifm"
+  type: FLOAT32
+  shape { dim: 1 dim: 3 dim: 3 dim: 2 }
+}
+# Add 100.0 is not to make divide by zero error in testing
```

Comment:
What if the `ifm` comes with value `-100.0`..?
`-100.0 + 100.0 = 0.0` !?

Reply:
ifm is random value, for float32,
- `compiler/common-artifacts/src/TestDataGenerator.cpp` generates with `std::normal_distribution<float> distrib(0, 2); // mean(0), stddev(2)`
- `compiler/luci-pass-value-py-test/test_luci_eval.py` generates 0 ~ 100